src/powfacpy/base_interface.py

```diff
@@ -692,6 +692,13 @@ def add_selected_variables_for_export(comres,results_variables_lists):
     comres.resultobj = results_variables_lists.result_objects
     comres.element = results_variables_lists.elements
     comres.variable = results_variables_lists.variables
+    if not results_variables_lists['variables'][0] == time_name:
+      results_variables_lists['result_objects'].insert(0,results_variables_lists['result_objects'][0])
+      results_variables_lists['elements'].insert(0,results_variables_lists['result_objects'][0])
+      results_variables_lists['variables'].insert(0,time_name)
+    comres.resultobj = results_variables_lists['result_objects']
+    comres.element = results_variables_lists['elements']
+    comres.variable = results_variables_lists['variables']
```

Comment:
What happens in this part? Not clear to me from the commit message. What is time_name?